src/jsonapi-vuex.js

```diff
@@ -359,6 +364,41 @@ const jsonapiModule = (api, conf = {}) => {
 
 // Helper methods
 
+const addJvHelpers = (obj) => {
+  // Add Utility functions to _jv child object
+  Object.assign(obj[jvtag], {
+    isRel(name) {
+      return get(obj, [jvtag, 'relationships'], {}).hasOwnProperty(name)
+    },
+    isAttr(name) {
+      return name !== jvtag && !this.isRel(name)
+    },
+  })
+  Object.defineProperty(obj[jvtag], 'rels', {
```

Comment:
worth adding comment why rels/attr added with defineProperty (maybe with link to current discussion)

Reply:
Good point - it's the kind of thing that looks like it needs tidy information, which will introduce a regression. Will update with a comment.